components/feature/addons/src/main/java/mozilla/components/feature/addons/migration/SupportedAddonsChecker.kt

```diff
@@ -154,6 +157,12 @@ internal class SupportedAddonsWorker(
                 addon.isSupported() && addon.isDisabledAsUnsupported()
             }
 
+            withContext(Dispatchers.Main) {
+                newSupportedAddons.map {
```

Comment:
nit: Should use `forEach` as we're not doing anything with the result of `map`.

Reply:
👍